docs/excel/excel-data-types-entity-card.md

```diff
@@ -27,7 +27,9 @@ The entity value [`properties`](/javascript/api/excel/excel.entitycellvalue#exce
 The following code snippet shows the JSON for an entity value with multiple data types nested within `properties`.
 
 > [!NOTE]
-> To see how to use this JSON in a complete code sample, visit the [OfficeDev/office-js-snippets](https://github.com/OfficeDev/office-js-snippets/blob/main/samples/excel/85-preview-apis/data-types-entity-values.yaml) repository.
+> To see how to use this JSON in a complete code sample, visit the [OfficeDev/office-js-snippets](https://github.com/OfficeDev/office-js-snippets/blob/prod/samples/excel/20-data-types/data-types-entity-values.yaml) repository.
```

Comment:
The previous examples say "Get Script Lab and try this sample". Here, the order is reversed. I like the pattern of App -> Sample and would recommend changing the three Script Lab examples on this page to match.

Reply:
Thanks for the review! Incorporated your feedback. Let me know if I misunderstood.